plugins/pastefromword/plugin.js

```diff
@@ -110,6 +110,13 @@
 				// this event.
 				isLazyLoad && evt.cancel();
 			}, null, null, 3 );
+
+			// Paste From Word Image:
+			// RTF clipboard is required for embeding images.
+			if ( CKEDITOR.plugins.clipboard.isCustomDataTypesSupported && configInlineImages ) {
+				editor.filter.allow( 'img[src]' );
```

Comment:
Actually in this case we don't want to force adding `img` to the allowed content list.

Instead we can do a bit more clever thing, and detect whether images are allowed. If not there's also no point in doing this logic. For example:

```js
editor.filter.check( 'img[src]' )
```